tests/test_workflow_init.py

```diff
@@ -72,3 +73,28 @@ def test_run_cleanup(tmp_path: Path, caplog: pytest.LogCaptureFixture):
     workflow = InitWorkflow(dpath_root=tmp_path)
     workflow.run_cleanup()
     assert f"Successfully initialized a dataset at {workflow.dpath_root}" in caplog.text
+
+
+def test_init_bids(tmp_path):
```

Comment:
Maybe add a check that the manifest has been created and is valid?

E.g. after the workflow is run
```
from nipoppy.tabular.manifest import Manifest
assert isinstance(workflow.manifest, Manifest)
```

Ideally also check that the manifest rows match what is expected

Reply:
And to increase test coverage we would need a test checking that the warning is raised in a BIDS dataset without session-level folders